docs/source/loggers/tensorboard_logger.md

```diff
@@ -0,0 +1,198 @@
+access to your cluster, you may not be able to access your Tensorboard logs after the training
+run is over. To solve this issue you can have your Tensorboard logs saved to some cloud storage
+somewhere, like Amazon S3. 
+
+<!-- I can't figure out how to use the ObjectStoreLogger pythonically, so punting on this -->
+<!-- To save your logs to S3, you will need create an  `ObjectStoreLogger` object, like so:
+
+```python
+from composer.loggers import ObjectStoreLogger
+from composer.utils import
+``` -->
+
+If you are using YAML configs you can get S3 logging by adding the following lines to your YAML file under loggers:
+
+```yaml
+loggers:
+  object_store:
+    object_store_hparams:
+      s3:
+        bucket: my-s3-bucket-name
+```
+Remember to replace `my-s3-bucket-name` with the name of your bucket.
+
+Putting this all together, your YAML file will look like:
+
+```yaml
+train_dataset:
+  mnist:
+    is_train: true
+    datadir: ./datasets/mnist
+    download: true
+val_dataset:
+  mnist:
+    is_train: false
+    datadir: ./datasets/mnist
+    download: true
+model:
+  mnist_classifier:
+    num_classes: 10
+max_duration: 5ep
+eval_interval: 1ep
+train_batch_size: 128
+eval_batch_size: 64
+loggers:
+  tensorboard:
+    log_dir: my_tensorboard_logs
+  object_store:
+    object_store_hparams:
+      s3:
+        bucket: my-s3-bucket-name
+```
+
+Once again you can run this job like so:
+
+```bash
+python examples/run_composer_trainer.py -f mnist.yaml
+```
+
+If all goes well: your loss and metric results will be logged Tensorboard log files, which will be written to `s3://my-s3-bucket-name/tensorboard_logs`
+The tensorboard logger will always name the directory inside your bucket "`tensorboard_logs`".
+
+See [these instructions](#viewing-your-results-from-s3) for viewing your results in the Tensorboard viewer.
+
+
+## Viewing your Results Locally
+If you saved your Tensorboard log files locally you can view them by starting a Tensorboard process and pointing it to the log directory you specified. To do this run the following at the command line:
+
+```bash
+tensorboard --logdir='./my_tensorboard_logs'
+```
+
+This will start a a Tensorboard process, which will write a message to stdout that looks something like:
+
+```bash
+TensorBoard 2.9.1 at http://localhost:6006/ (Press CTRL+C to quit)
+```
+
+Open the URL in your browser to access the Tensorboard viewer, which should look something like this:
+![tb_screenshot](./tensorboard_screenshot.png)
+
+Enjoy viewing your metrics!
+
+
+
+## Viewing your Results from S3
+To view your logs that are saved to S3, you first need to copy them from S3 to your local system. You can do so by using the AWS cli, which you can download [here](https://aws.amazon.com/cli/). Once you have that installed, you can run the following commands:
+
+```bash
+mkdir my_tensorboard_logs
+aws s3 cp s3://my-s3-bucket-name/tensorboard_logs ./my_tensorboard_logs --recursive
```

Comment:
If you have the AWS CLI installed and are logged in with `aws sso login` then you don't need to copy the files locally. You should be able to pass the S3 path directly to the tensorboard command like:

```
 tensorboard --logdir=s3://mosaicml-internal-checkpoints-bert/tensorboard_logs/bert-tutorial-71gw/
```
and it will work.

Reply:
oh nice! I need to test that myself! done